src/components/Tabs/Tabs.tsx

```diff
@@ -79,7 +75,7 @@ export const Tabs = ({
   );
 
   const tabIds = useMemo(
-    () => tabs.map((tab) => tab.props['aria-labelledby'] || getUID(tab)),
+    () => tabs.map((tab) => tab.props.id || getUID(tab)),
```

Comment:
Finally realized that it makes no sense to use `aria-labelledby` here as the id for a tab...

Think 🤔 about it like this. If we have

```jsx
<h2 id="heading">Stuff!</h2>

<Tabs>
  <Tab aria-labelledby="heading" title="Thing 1" />
</Tabs>
```

we'll end up with

```html
<h2 id="heading">Stuff!</h2>

<ul>
  <li id="heading">Thing 1</li>
</ul>
```

The ids are duplicated, and the logic here is clearly wrong.